free-1.md

```diff
@@ -57,12 +159,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+If I was starting from complete scratch, the game design aspects of this
+project would eat up a solid amount of time. That being said, since I have
+started this project some time ago, I already have a lot of ideas and 
+beta code for displaying grpahics, getting input, etc. At this point,
+although I will still need to work on how the parsed DSL is interpreted
+within the game environment, a lot of the typical game aspects have been coded.
+Because of this, I think a vast majority of my time will be spent directly
+engaging with the language aspects of this project.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The overall scoping of this game is far to ambitious for a CS111 project. In
+fact, after I graduate, I plan to see this idea to some type of completion and
+would like to even distribute it. That being said, I think by looking at this
+project as a way to develop a prototype of some early tutorial levels will
+make it a very reasonable project.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This project might be a good idea because I am essentially developing an entire
+external DSL from scratch that will be interpreted by game code! I will
+certainly learn a lot by undertaking this project plus I'm super excited
+about it.
+
+On this other hand, this project might not be a good idea because it is certain
+on the larger project that is likely too ambitious even if carefully scoped.
```

Comment:
I think this is the biggest problem here as well. It seems as if the scoping on this project would have to be done _very_ deliberately.